.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -49,25 +49,21 @@ body:
       validations:
           required: false
 
-    - type: dropdown
+    - type: checkboxes
       id: existing
       attributes:
           label: Please confirm that you have searched existing issues in the repo.
           description: You can do this by searching https://github.com/WordPress/gutenberg/issues and making sure the bug is not related to another plugin.
-          multiple: true
           options:
-              - 'Yes'
-              - 'No'
+              - label: 'Yes'
       validations:
```

Comment:
It seems that if we want a checkbox to be required, we need to define it in the `options` field, not the `validations` field.

See this code sample: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

Reply:
Ok, it should be good now. You can actually preview it here https://github.com/WordPress/gutenberg/blob/dc41b4fd10d6828f3985cffecfc52143d1011646/.github/ISSUE_TEMPLATE/Bug_report.yml